zerver/views/user_settings.py

```diff
@@ -229,3 +230,18 @@ def change_enter_sends(request: HttpRequest, user_profile: UserProfile,
                        enter_sends: bool=REQ(validator=check_bool)) -> HttpResponse:
     do_change_enter_sends(user_profile, enter_sends)
     return json_success()
+
+@human_users_only
+@has_request_variables
+def export(request: HttpRequest, user_profile: UserProfile) -> HttpResponse:
+    user_id = user_profile.id
+    data = {'user_profile_id': user_id, 'type': 'export_user_messages'}
+    if not UserDataExport.export_running(user_id):
+        UserDataExport.log_new_export(user_id)
+        queue_json_publish('deferred_work', data)
+        msg = 'Your export has been queued!'
+
+    else:
+        msg = 'An export is already queued for you.  You will need to wait until it completes.'
```

Comment:
These should get tagged for translation with `_()`.  Also, check how `json_success` handles arguments in common use; I think you actually just want these two code paths to be e.g. `json_success(_("Your export has been queued"))`